packages/lit-element/package.json

```diff
@@ -39,30 +39,31 @@
   },
   "author": "The Polymer Authors",
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
```

Comment:
Any reason why `package.json` changed?

Reply:
Oh, this also looks like why CI is failing (`package.json` & `package-lock.json` disagreement)

Reply:
Fixed.

Reply:
I'm still seeing the package.json changes. Why are those in these PR?

Reply:
I sync'd the package.json with lit-html so that we can test multiple browsers.